runtime/mainnet/src/config/governance.rs

```diff
@@ -0,0 +1,387 @@
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_collective::Config<CouncilCollective>>::KillOrigin>(
+				),
+				TypeId::of::<EnsureRoot<AccountId>>(),
+			);
+		}
+
+		#[test]
+		fn number_of_councilors_is_limited() {
+			assert_eq!(
+				<<Runtime as pallet_collective::Config<CouncilCollective>>::MaxMembers as Get<
+					u32,
+				>>::get(),
+				100,
+			);
+		}
+
+		#[test]
+		fn proposal_weight_is_limited() {
+			assert_eq!(
+				TypeId::of::<
+					<Runtime as pallet_collective::Config<CouncilCollective>>::MaxProposalWeight,
+				>(),
+				TypeId::of::<MaxProposalWeight>(),
+			);
+
+			assert_eq!(
+				<<Runtime as pallet_collective::Config<CouncilCollective>>::MaxProposalWeight as Get<Weight>>::get(),
+				sp_runtime::Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block,
+			);
+		}
+
+		#[test]
+		fn authorize_upgrade_does_not_saturate_weight_limit() {
+			let authorize_upgrade_weight =
+				<<Runtime as frame_system::Config>::SystemWeightInfo>::authorize_upgrade();
+			let max_proposal_weight = <<Runtime as pallet_collective::Config<CouncilCollective>>::MaxProposalWeight as Get<Weight>>::get();
+			assert!(authorize_upgrade_weight.all_lt(max_proposal_weight));
+		}
+
+		#[test]
+		fn number_of_proposals_is_limited() {
+			assert_eq!(
+				<<Runtime as pallet_collective::Config<CouncilCollective>>::MaxProposals as Get<
+					u32,
+				>>::get(),
+				100,
+			);
+		}
+
+		#[test]
+		fn motion_duration_is_7_days() {
+			assert_eq!(
+				TypeId::of::<
+					<Runtime as pallet_collective::Config<CouncilCollective>>::MotionDuration,
+				>(),
+				TypeId::of::<CouncilMotionDuration>(),
+			);
+
+			assert_eq!(
+				<<Runtime as pallet_collective::Config<CouncilCollective>>::MotionDuration as Get<BlockNumber>>::get(),
+				7 * DAYS,
+			);
+		}
+
+		#[test]
+		fn proposals_are_runtime_calls() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_collective::Config<CouncilCollective>>::Proposal>(),
+				TypeId::of::<RuntimeCall>(),
+			);
+		}
+
+		#[test]
+		fn set_members_origin_ensures_root() {
+			assert_eq!(
+				TypeId::of::<
+					<Runtime as pallet_collective::Config<CouncilCollective>>::SetMembersOrigin,
+				>(),
+				TypeId::of::<EnsureRoot<AccountId>>(),
+			);
+		}
+
+		#[test]
+		fn default_weights_are_not_used() {
+			assert_ne!(
+				TypeId::of::<<Runtime as pallet_collective::Config<CouncilCollective>>::WeightInfo>(
+				),
+				TypeId::of::<()>(),
+			);
+		}
+	}
+
+	mod membership {
+		use super::*;
+
+		#[test]
+		fn add_origin_requires_unanimous_vote() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::AddOrigin>(
+				),
+				TypeId::of::<UnanimousCouncilVote>(),
+			);
+
+			assert_eq!(
+				TypeId::of::<UnanimousCouncilVote>(),
+				TypeId::of::<
+					EitherOfDiverse<
+						EnsureRoot<AccountId>,
+						EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 1>,
+					>,
+				>(),
+			);
+		}
+
+		#[test]
+		fn number_of_members_is_100() {
```

Comment:
I know I gave a recommendation on this one before, but with fresh eyes this is misleading as well
```suggestion
		fn max_members_is_100() {
```